std_msgs/string type
robot_msgs/JointTrajPoint start
robot_msgs/JointTrajPoint joint_goal
robot_msgs/Pose cartesian_goal
---
robot_msgs/JointTraj traj
